docs/database/schema.sql

```diff
@@ -51,15 +51,16 @@ CREATE UNIQUE INDEX connector_property_connector_id_property_index_uindex ON con
 -- "twitter")
 CREATE TABLE action
 (
-    id SERIAL PRIMARY KEY NOT NULL,
+    id INTEGER DEFAULT nextval('action_id_seq'::regclass) PRIMARY KEY NOT NULL,
     connector_id INTEGER NOT NULL,
     name TEXT NOT NULL,
     description TEXT,
     tags TEXT[],
+    camel_connector_maven_gav TEXT NOT NULL,
```

Comment:
generated png still shows maven_gav (which confused me a bit first). Thats much better ;-). Maybe one could omit "maven", as "gav" is probably expressive enough.